### PR TITLE
cso metrics

### PR DESCRIPTION
**Pull request by xithan**

### CODE REVIEW

File: db/migrate_cards/20160514205559_add_cso_metrics.rb [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      76 => ['Annual CO2 Emissions', nil],
+      77 => ['Gross Revenue', nil],
+
+      80 => ['Cumulative CO2 Emissions Context-Free Score', '86/89'],
+      81 => ['Annual CO2 Emissions Context-Based Relative Score', '(76/75)/95'],
+      82 => ['Annual CO2 Emissions Context-Based Absolute Score', '76/91'],
+      83 => ['Cumulative CO2 Emissions Context-Based Absolute Score', '86/92'],
+
+      85 => ['CO2 Emissions Relative to Gross Revenue', '76/77'],
+      86 => ['Cumulative CO2 Emissions', '#85'],
```
Comment (ethn) [inline db/migrate_cards/20160514205559_add_cso_metrics.rb:24]: needs to be #76.  (and needs to sum them all)

File: db/migrate_cards/20160514205559_add_cso_metrics.rb [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+      88 => ['Maximum Allowable Annual CO2 Emissions', '$76*71'],
+      89 => ['Maximum Allowable Cumulative CO2 Emissions', '#88'],
```
Comment (ethn) [inline db/migrate_cards/20160514205559_add_cso_metrics.rb:27]: should also sum

File: db/migrate_cards/20160514205559_add_cso_metrics.rb [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+      91 => ['Annual CO2 Emission Targets', '95*75'],
+      92 => ['Cumulative CO2 Emission Targets', '#91'],
+
+      94 => ['Maximum Allowable Annual CO2 Emissions per C2GDP', '$76/$75*71'],
+      95 => ['Maximum Allowable Annual CO2 Emissions per C2GDP - ' \
+             'Adjusted per OECD norm',
+             '(70*1000000000)/' \
+             '(($70*1000000000-$88)/($72-$75)*(72-75)*71+94*75)*94']
```
Comment (ethn) [inline db/migrate_cards/20160514205559_add_cso_metrics.rb:36]: the "about" section here will be tough to write :)

Comment (ethn) [inline db/migrate_cards/20160514205559_add_cso_metrics.rb:36]: can we add comments in the wolfram formula?

File: db/migrate_cards/20160514205559_add_cso_metrics.rb [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+      85 => ['CO2 Emissions Relative to Gross Revenue', '76/77'],
+      86 => ['Cumulative CO2 Emissions', '#85'],
+
+      88 => ['Maximum Allowable Annual CO2 Emissions', '$76*71'],
+      89 => ['Maximum Allowable Cumulative CO2 Emissions', '#88'],
+
+      91 => ['Annual CO2 Emission Targets', '95*75'],
+      92 => ['Cumulative CO2 Emission Targets', '#91'],
+
+      94 => ['Maximum Allowable Annual CO2 Emissions per C2GDP', '$76/$75*71'],
```
Comment (ethn) [inline db/migrate_cards/20160514205559_add_cso_metrics.rb:32]: this is what I have in the spreadsheet, but rearranging it to ($76*71)/$75 might make it easier to get your head around.

File: db/migrate_cards/20160514205559_add_cso_metrics.rb [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+      83 => ['Cumulative CO2 Emissions Context-Based Absolute Score', '86/92'],
+
+      85 => ['CO2 Emissions Relative to Gross Revenue', '76/77'],
+      86 => ['Cumulative CO2 Emissions', '#85'],
+
+      88 => ['Maximum Allowable Annual CO2 Emissions', '$76*71'],
+      89 => ['Maximum Allowable Cumulative CO2 Emissions', '#88'],
+
+      91 => ['Annual CO2 Emission Targets', '95*75'],
+      92 => ['Cumulative CO2 Emission Targets', '#91'],
```
Comment (ethn) [inline db/migrate_cards/20160514205559_add_cso_metrics.rb:30]: this one needs sum, too.  I guess all the # ones do.

File: db/migrate_cards/20160514205559_add_cso_metrics.rb (hunk 3 above)
Comment (xithan) [inline db/migrate_cards/20160514205559_add_cso_metrics.rb:36]: comments in nests should be fine

**Pull request merged**